src/opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -215,6 +215,9 @@ namespace Opm {
         using GC = ParserKeywords::GCOMPIDX;
         if (deck.hasKeyword<GC>())
             this->m_gas_comp_index = deck.getKeyword<GC>().getRecord(0).getItem<GC::GAS_COMPONENT_INDEX>().get<int>(0);
+        else
+            // Otherwise valgrind complains with MPI
+            this->m_gas_comp_index = 77;
```

Comment:
Maybe rather set it in the `TableManager.hpp` file - a bit surprising this does not hit the other members of the `TableManager` class

Reply:
Well they either must be set unconditionally or are not pods and hence default initialized.